cdc/sinkv2/eventsink/mq/worker.go

```diff
@@ -169,7 +169,7 @@ func (w *worker) asyncSend(
 	for key, events := range partitionedRows {
 		for _, event := range events {
 			// Skip this event when the table is stopping.
-			if event.TableStatus.Load() == pipeline.TableStateStopping {
+			if event.SinkState.Load() == state.TableSinkStopping {
```

Comment:
Can you wrap `event.SinkState.Load()` in a method, probably called `event.GetTableState()`?

Reply:
Good suggestion!